src/electron/release_linux_action.sh

```diff
@@ -18,6 +18,37 @@
 #  - prod Sentry keys are used
 #  - auto-updates are configured
 
+function usage () {
+  echo "$0 [-s stagingPercentage]" 1>&2
+  echo "  -s: The staged rollout percentage for this release.  Must be in the interval (0, 100].  Defaults to 100" 1>&2
+  echo "  -h: this help message" 1>&2
+  echo 1>&2
+  echo "Examples:" 1>&2
+  echo "Releases the beta of Linux version 1.2.3 to 10% of users listening on the beta channel" 1>&2
+  echo "TRAVIS_TAG=linux-v1.2.3-beta  $0 -s 10" 1>&2
+  exit 1
+}
+
+STAGING_PERCENTAGE=100
+while getopts s:? opt; do
+  case $opt in
+    s) STAGING_PERCENTAGE=$OPTARG ;;
+    *) usage ;;
+  esac
+done
+
+if ((STAGING_PERCENTAGE <= 0)) || ((STAGING_PERCENTAGE > 100)); then
+  echo "Staging percentage must be greater than 0 and no more than 100"
+  exit 1
+fi
+
+TAG=$(scripts/tag.sh linux)
+if [[ $TAG =~ ^.*-beta$ ]]; then
+  export CHANNEL="beta"
```

Comment:
I couldn't find it in the documentation - does electron use this variable? If it does, maybe add a comment; otherwise, no need to export.